### PR TITLE
Remove the VPN feature tour

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
## Description

This removes the VPN tour.

## Reference

VPN-2798

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: src/qmake/sources.pri [hunk 1]
```diff
@@ -108,7 +108,6 @@ SOURCES += \
         models/subscriptiondata.cpp \
         models/supportcategorymodel.cpp \
         models/user.cpp \
-        models/whatsnewmodel.cpp \
```
Comment (bakulf) [inline src/qmake/sources.pri:111]: You have to remove these files from `src/cmake/source.*`

File: src/featureslist.h [hunk 2]
```diff
@@ -164,49 +139,27 @@ FEATURE_SIMPLE(mobileOnboarding,     // Feature ID
...
-    "https://support.mozilla.org/kb/"
-    "protect-your-container-tabs-mozilla-vpn?utm_source=mozilla.vpn."
-    "client&utm_medium=in-app-link&utm_content=whats-new-panel&utm_"
-    "campaign=vpn-better-together",  // link URL
-    "2.7",
-    false,          // Can be flipped on
-    false,          // Can be flipped off
-    QStringList(),  // feature dependencies
-    FeatureCallback_multiAccountContainers)
+FEATURE_SIMPLE(multiAccountContainers,      // Feature ID
```
Comment (bakulf) [inline src/featureslist.h:142]: You can remove this feature. And also this https://searchfox.org/mozilla-vpn-client/source/src/featureslistcallback.h#85

File: src/featureslist.h [hunk 3]
```diff
@@ -58,54 +58,29 @@ FEATURE_SIMPLE(captivePortal,     // Feature ID
...
-    L18nStrings::WhatsNewReleaseNotesEasyProfileDescription,  // LongDescr
-    "qrc:/ui/resources/features/easy-profile-preview",        // ImagePath
-    "",                                                       // IconPath
-    "",                                                       // link URL
-    "2.9",
-    false,          // Can be flipped on
-    false,          // Can be flipped off
-    QStringList(),  // feature dependencies
-    FeatureCallback_true)
+FEATURE_SIMPLE(easyProfile,     // Feature ID
```
Comment (bakulf) [inline src/featureslist.h:77]: Remove this feature

File: src/featureslist.h [hunk 4]
```diff
@@ -58,54 +58,29 @@ FEATURE_SIMPLE(captivePortal,     // Feature ID
...
-    L18nStrings::WhatsNewReleaseNotesConnectionInfoDescription,  // LongDescr
-    "qrc:/ui/resources/features/connection-info-preview.png",    // ImagePath
-    "qrc:/ui/resources/features/connection-info-icon.svg",       // IconPath
-    "",                                                          // link URL
-    "2.8",                                                       // released
-    true,           // Can be flipped on
-    false,          // Can be flipped off
-    QStringList(),  // feature dependencies
-    FeatureCallback_true)
+FEATURE_SIMPLE(connectionInfo,     // Feature ID
```
Comment (bakulf) [inline src/featureslist.h:61]: Remove this feature

**Review by flodolo — APPROVED** [review]

**Pull request merged**